test/cases/testdata/cryptohmacsha256/test-cryptohmacsha256.yaml

```diff
@@ -0,0 +1,14 @@
+cases:
+- note: cryptohmacsha256/crypto.hmac.sha256
+  query: data.test.p = x
+  modules:
+  - |
+    package test
+    
+    p[mac] {
+      mac := crypto.hmac.sha256(input.message, input.key)
+    }
+  input: {"message": "foo", "key": "bar"}
+  want_result:
```

Comment:
It would be good with at least one test providing unicode characters in string and/or key.

Reply:
Looking into this.

Reply:
New tests are great!